src/Point.ts

```diff
@@ -0,0 +1,27 @@
+export class Point {
+  x: number;
+  y: number;
+
+  constructor(x: number = 0, y: number = 0) {
```

Comment:
The task says to create default constructor and overloaded constructor, so there probably should be a declaration without any parameters (for default constructor).